nemo-seahorse does not build on debian-sid

nemo-seahorse does not build on debian-sid

```
Running compile:
Working directory:  /build/nemo-seahorse-5.6.0/obj-x86_64-linux-gnu/meson-private/tmpyh0klflh
Command line:  cc /build/nemo-seahorse-5.6.0/obj-x86_64-linux-gnu/meson-private/tmpyh0klflh/testfile.c -o /build/nemo-seahorse-5.6.0/obj-x86_64-linux-gnu/meson-private/tmpyh0klflh/output.exe -g -O2 -ffile-prefix-map=/build/nemo-seahorse-5.6.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64 -O0 -Wl,--start-group -lgpgme -Wl,--end-group -Wl,--allow-shlib-undefined -Wl,-z,relro -Wl,-z,now 

Code:
 int main(void) { return 0; }

Compiler stdout:
 
Compiler stderr:
 
Library gpgme found: YES
Program gpgme-config found: NO

../meson.build:64:0: ERROR: Program 'gpgme-config' not found or not executable
dh_auto_configure: error: cd obj-x86_64-linux-gnu && LC_ALL=C.UTF-8 meson setup .. --wrap-mode=nodownload --buildtype=plain --prefix=/usr --sysconfdir=/etc --localstatedir=/var --libdir=lib/x86_64-linux-gnu returned exit code 1
make: *** [debian/rules:6: build] Error 255
dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2

```


```
 $ apt-file find gpgme-config
librust-gpgme-sys-dev: /usr/share/cargo/registry/gpgme-sys-0.10.0/debian/patches/workaround-gpgme-config-removal.patch
```


Thanks...

--- a/nemo-seahorse/meson.build
+++ b/nemo-seahorse/meson.build
@@ -1,7 +1,7 @@
 project('nemo-seahorse',
   'c',
   version: '5.6.0',
-  meson_version: '>=0.49.0'
+  meson_version: '>=0.51.0'
 )
 
 project_url = 'https://github.com/linuxmint/nemo-extensions'
@@ -60,14 +60,7 @@ if get_option('gpg-check')
     endif
 endif
 
-libgpgme = meson.get_compiler('c').find_library('gpgme')
-gpgme_config = find_program('gpgme-config')
-gpgme_ver = run_command(gpgme_config, '--version').stdout().strip()
-gpgme_ver_required = '>=1.2.0'
-if not gpgme_ver.version_compare(gpgme_ver_required)
-    error('GPGME version @0@ was found, @1@ is required.'.format(
-        gpgme_ver, gpgme_ver_required))
-endif
+libgpgme = dependency('gpgme', version: '>=1.2.0')
 
 ################################################################################
 # Generic stuff

